Microsoft.Azure.Cosmos.Samples/Usage/BulkSupport/Program.cs

```
@@ -149,7 +152,67 @@ private static async Task CreateItemsConcurrentlyAsync(Container container)
+                "select * from T where IS_DEFINED(T.other)",
+                requestOptions: new QueryRequestOptions()
+                {
+                    MaxBufferedItemCount = 0,
+                    MaxConcurrency = 1,
+                    MaxItemCount = 100
+                });
+
+            while (queryOfItemsToUpdate.HasMoreResults)
+            {
+                FeedResponse<JObject> items = await queryOfItemsToUpdate.ReadNextAsync();
+                List<Task> tasks = new List<Task>(1000);
+                foreach (JObject item in items)
+                {
+                    tasks.Add(Program.RemoveItemProperty(container, item));
+                }
+
+                await Task.WhenAll(tasks);
+            }
+
+            Console.WriteLine($"All items updated to remove property 'other'");
+        }
+
+        private static async Task RemoveItemProperty(
+            Container container,
+            JObject item)
+        {
+            ItemRequestOptions itemRequestOptions = new ItemRequestOptions()
+            {
+                EnableContentResponseOnWrite = false,
+            };
+
+            while (true)
+            {
+                item.Remove("other");
+                string id = item["id"].Value<string>();
+                string pk = item["pk"].Value<string>();
+                itemRequestOptions.IfMatchEtag = item["_etag"].Value<string>();
+                try
+                {
+                    await container.ReplaceItemAsync<JObject>(item, id, new PartitionKey(pk));
+                    return;
+                }
+                catch (CosmosException ex)
+                {
+                    Console.WriteLine($"Replace item failed at {DateTime.UtcNow} with id:{id}; pk:{pk}; Excepion: {ex}");
+                    // The item was updated after the query. Read the latest item and try update again.
+                    if (ex.StatusCode == HttpStatusCode.Conflict)
```

Silently eating non-conflict ones. Prefer conditional catch